keanu-python/keanu/vertex/label.py

```diff
@@ -0,0 +1,14 @@
+from py4j.java_gateway import java_import
+from keanu.base import JavaObjectWrapper
+from keanu.context import KeanuContext
+
+k = KeanuContext()
+
+java_import(k.jvm_view(), "io.improbable.keanu.vertices.VertexLabel")
+
+
+class _VertexLabel(JavaObjectWrapper):
```

Comment:
Why is the class prefixed with an underscore?

Reply:
because a user shouldn't use it